lib/services/livesync/debug-livesync-service.ts

```diff
@@ -66,7 +66,7 @@ export class DebugLiveSyncService extends LiveSyncService implements IDebugLiveS
 		const buildConfig: IBuildConfig = _.merge({ buildForDevice: !deviceAppData.device.isEmulator }, deployOptions);
 		debugData.pathToAppPackage = this.$platformService.lastOutputPath(debugService.platform, buildConfig, projectData);
 
-		this.printDebugInformation(await debugService.debug<string[]>(debugData, debugOptions));
+		this.printDebugInformation([await debugService.debug(debugData, debugOptions)]);
```

Comment:
we always call `printDebugInformation` by manually converting some string to array, can't we change the `printDebugInformation` method to has `string` for parameter instead of `string[]`